lib/racecar/cli.rb

```diff
@@ -45,7 +53,26 @@ def self.main(args)
       end
 
       $stderr.puts "=> Wrooooom!"
-      $stderr.puts "=> Ctrl-C to shutdown consumer"
+
+      if daemonize
+        daemon = Daemon.new(File.expand_path(Racecar.config.pidfile))
+
+        daemon.check_pid
+
+        $stderr.puts "=> Starting background process"
+        $stderr.puts "=> Writing PID to #{daemon.pidfile}"
+
+        if Racecar.config.logfile.nil?
+          daemon.suppress_output
+        else
+          daemon.redirect_output(Racecar.config.logfile)
+        end
+
+        daemon.daemonize!
+        daemon.write_pid
+      else
+        $stderr.puts "=> Ctrl-C to shutdown consumer"
```

Comment:
Operator information is printed to stderr while runtime logging happens on stdout. That way, doing `racecar MyConsumer > racecar.log` would still show important info in the terminal.